app/src/main/java/com/animal_villa/dto/Character.kt

```diff
@@ -1,4 +1,23 @@
 package com.animal_villa.dto
+/**
+ * A class Character is noun class Data Transfer Object.
+ * @property name the name for every Character : String
+ * @property age the age for every Character : Int
+ * @property job the job for every Character : String
+ */
+data class Character(var name : String, var age : Int, var job : String) {
```

Comment:
Implemented data class instead of class as it is a DTO with the purpose to hold data. https://kotlinlang.org/docs/data-classes.html
Initialized constructor variables accorrding to conventions, class headers https://kotlinlang.org/docs/coding-conventions.html#class-headers